### PR TITLE
docs: Add OpenAIAssistantV2Runnable docstrings

### PR DESCRIPTION
**Pull request by hmnfalahi**
- **Description:** add/improve docstrings of OpenAIAssistantV2Runnable
- **Issue:** the issue #21983

### CODE REVIEW

File: libs/community/langchain_community/agents/openai_assistant/base.py [hunk 1]
```diff
@@ -263,37 +292,21 @@ def create_assistant(
...
-                instructions: Additional run instructions.
-                model: Override Assistant model for this run.
-                tools: Override Assistant tools for this run.
-                tool_resources: Override Assistant tool resources for this run (v2 API).
-                run_metadata: Metadata to associate with new run.
-            config: Runnable config:
-
-        Return:
-            If self.as_agent, will return
+            input (dict): Input dictionary containing user message, thread ID, etc.
```
Comment (eyurtsev) [inline libs/community/langchain_community/agents/openai_assistant/base.py:298]: why was this deleted?

File: libs/community/scripts/check_pydantic.sh [hunk 2]
```diff
@@ -20,7 +20,7 @@ count=$(git grep -E '(@root_validator)|(@validator)|(@field_validator)|(@pre_ini
 # PRs that increase the current count will not be accepted.
 # PRs that decrease update the code in the repository
 # and allow decreasing the count of are welcome!
-current_count=127
+current_count=0
```
Comment (eyurtsev) [inline libs/community/scripts/check_pydantic.sh:23]:
```suggestion
current_count=127
```

File: libs/community/scripts/check_pydantic.sh [hunk 3]
```diff
@@ -41,7 +41,7 @@ count=$(git grep -E '^import pydantic_settings|^from pydantic_settings' -- "*.py
 # PRs that increase the current count will not be accepted.
 # PRs that decrease update the code in the repository
 # and allow decreasing the count of are welcome!
-current_count=8
+current_count=0
```
Comment (eyurtsev) [inline libs/community/scripts/check_pydantic.sh:44]:
```suggestion
current_count=8
```

File: libs/community/langchain_community/agents/openai_assistant/base.py (hunk 1 above)
Comment (hmnfalahi) [reply libs/community/langchain_community/agents/openai_assistant/base.py]: mistake, it's fixed now

**Review by ccurme — APPROVED** [review]
